src/utils/chains.ts

```diff
@@ -71,7 +70,7 @@ export const isRouteEnabled = (route: string, chain?: ChainInfo) => {
 }
 
 export const getLatestSafeVersion = (chain: ChainInfo | undefined): SafeVersion => {
-  const latestSafeVersion = chain && hasFeature(chain, FEATURES.SAFE_141) ? LATEST_SAFE_VERSION : FALLBACK_SAFE_VERSION
+  const latestSafeVersion = chain?.recommendedMasterCopyVersion || LATEST_SAFE_VERSION
```

Comment:
What if a network doesn't have 1.4.1? Previously we would use `FALLBACK_SAFE_VERSION` which defaulted to 1.3.0 but now this won't be the case anymore.

Reply:
It will fail somewhere down the line where it gets an actual deployment. 1.3.0 is also not guaranteed to exist on all chains.